README.md

```diff
@@ -407,23 +407,26 @@ Result:
 1. Install Kalix CLI
    https://docs.kalix.io/setting-up/index.html#_1_install_the_kalix_cli
 2. Kalix CLI
-1. Register (FREE)
+   1. Register (FREE)
    ```
    kalix auth signup
    ```
    **Note**: Following command will open a browser where registration information can be filled in<br>
```

Comment:
That's what I meant.  It's not the following command.  It's "the command above..."